stable/postgresql/templates/_helpers.tpl

```diff
@@ -40,9 +40,9 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 Return the appropriate apiVersion for networkpolicy.
 */}}
 {{- define "postgresql.networkPolicy.apiVersion" -}}
-{{- if semverCompare ">=1.4-0, <1.7-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.4-0, <1.7-0" (default .Capabilities.KubeVersion.GitVersion .Capabilities.KubeVersion.Version) -}}
```

Comment:
Can you try `String`? The same in the rest of cases
```suggestion
{{- if semverCompare ">=1.4-0, <1.7-0" (default .Capabilities.KubeVersion.GitVersion .Capabilities.KubeVersion.String) -}}
```

Reply:
Ahh thanks for finding that. Updated to what you suggested and it works in Helm 2.x and Helm 3.x